packages/cactus-plugin-consortium-manual/src/main/typescript/plugin-consortium-manual.ts

```diff
@@ -216,4 +227,46 @@ export class PluginConsortiumManual
   public getAspect(): PluginAspect {
     return PluginAspect.CONSORTIUM;
   }
+
+  public async createJws(): Promise<JWSGeneral> {
```

Comment:
@AzaharaC Please name this method to match the endpoint that it is the backing implementation of, e.g. something like `getNodeJws()`